### DISCUSSION TITLE
Possibility to delete single pages of a pdf

### DISCUSSION THREAD

**Discussion by Skylinar** [jonaswinkler/paperless-ng#1214] [Feature Requests]
Several scanners have the ability to scan double-sided. Since there are multi-page documents where not all pages are printed on both sides, there are usually blank pages in the pdf. It would be wonderful if it would be possible to delete individual pages of the pdf.

**Comment by oeiber** [comment]
The ability to delete blank pages as well as functionallity to reorder pages would be nice.

**Reply by TobenderZephyr** [reply]
Please see the Pinned Issue #335. There has already been work done on it.
